task/mongo-tasks.js

```diff
@@ -251,7 +813,75 @@ async function task_1_18(db) {
  *       - do not hesitate to "ensureIndex" in "before" function at the top if needed https://docs.mongodb.com/manual/reference/method/db.collection.ensureIndex/
  */
 async function task_1_19(db) {
-    throw new Error("Not implemented");
+    let result = await db.collection('orders').aggregate([
+        {
+          '$project': {
+            '_id': 0,
+            'CustomerID': 1, 
+            'OrderID': 1
+          }
+        }, {
+          '$lookup': {
+            'from': 'order-details', 
+            'localField': 'OrderID', 
+            'foreignField': 'OrderID', 
+            'as': 'T1'
+          }
+        }, {
+          '$unwind': {
+            'path': '$T1'
+          }
+        }, {
+          '$group': {
+            '_id': '$CustomerID', 
+            'total': {
+              '$sum': {
+                '$multiply': [
+                  {
+                    '$subtract': [
+                      '$T1.UnitPrice', 0
+                    ]
```

Comment:
why you need the `'$subtract'` here? looks like redundant part of the query

Reply:
Yes, it is. The first time I was comparing the query result with the MySQL test file and doing some manipulation with 'discount' but after checking MongoDB test file and found out that they are different.